scripts/rabbitmq-script-wrapper

```diff
@@ -83,10 +77,21 @@ exec_script_as_rabbitmq() {
 exec_script_as_root() {
   if [ -x /sbin/runuser ]
   then
-    exec /sbin/runuser -s /bin/sh -c "/usr/lib/rabbitmq/bin/$SCRIPT $CMDLINE" rabbitmq
+    # TODO:
+    # At some point all of the RabbitMQ supported distributions will be using
+    # the util-linux version of /sbin/runuser, as it has been removed from GNU
+    # coreutils as of 2012. At that point the first clause of the following
+    # if statement can become the only statement used and the if/then
+    # removed
+    if /sbin/runuser --version | grep -qF util-linux
+    then
+        exec /sbin/runuser -u rabbitmq -- "/usr/lib/rabbitmq/bin/$SCRIPT" "$@"
```

Comment:
`"$@"` is always empty here because `exec_script_as_root`, called on line 35, isn't passed `"$@"`. Therefore any use of `rabbitmqctl` as root show the usage and exits with 64.

I will commit the fix I just tried. Everything is working as expected after that.

Reply:
@dumbbell thanks for catching that ... I forgot to run `shellcheck` which would have found that:

```
$ shellcheck scripts/rabbitmq-script-wrapper

In scripts/rabbitmq-script-wrapper line 35:
    exec_script_as_root
    ^-- SC2119: Use exec_script_as_root "$@" if function's $1 should mean script's $1.


In scripts/rabbitmq-script-wrapper line 77:
exec_script_as_root() {
^-- SC2120: exec_script_as_root references arguments, but none are ever passed.
```